package-lock.json

```diff
@@ -1,8 +1,19712 @@
 {
   "name": "@freecodecamp/freecodecamp",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
lockfile Version 2 is used in `npm` versions 7 and higher - our codebase officially supports `npm` major version 6. The changes to this file should be removed from your commit.